blockchain/blockchain.go

```diff
@@ -1735,6 +1739,10 @@ type insertStats struct {
 	startTime                  mclock.AbsTime
 }
 
+// statsReportLimit is the time limit during import and export after which we always print
+// always print out progress. This avoids the user wondering what's going on.
+const statsReportLimit = 8 * time.Second
```

Comment:
We already have `log.StatsReportLimit` having the same comment.

Reply:
Thanks, I revised it! https://github.com/klaytn/klaytn/pull/743/commits/1f3a296db36006df4a4e251dc87a914c11f7348f